File: 0009-palindrome-number/solution.c
bool isPalindrome(int x) {
    long rev=0,rem=0,n=x;
    while(n>0)
    {
        rem=n%10;
        rev=(rev*10)+rem;
        n/=10;
    }
    if(x==rev)
    return 1;
    else
    return 0;
}

File: 0009-palindrome-number/solution.java
class Solution {
    public int reverse(int x){
        int rev = 0;
        while(x != 0){
            rev = (rev * 10) + (x % 10);
            x /= 10;
        }
        return rev;
    }
    public boolean isPalindrome(int x) {
        if( x < 0 ) return false;
        else if (reverse(x) == x) return true;
        else return false;
    }
}
